content/post/announcing-pga.md

```diff
@@ -57,6 +57,22 @@ Similarly, the number of lines of code show the size of Public Git Archive.
 
 ![lines of code of datasets](/post/announcing-pga/loc.png)
 
+## Comparing PGA to GHTorrent and the BigQuery GitHub dataset
+
+Many people asked us already how this dataset compares to two related
+projects: GHTorrent and BigQuery.
```

Comment:
Would not it be more accurate to call it here and in title as "GitHub public dataset in BigQuery"?

Reply:
The official name seems to be "GitHub Data"; I am happy with all the three variants, they are impossible to confuse anyway.